portfolio/src/main/java/com/google/sps/data/AuthData.java

```diff
@@ -0,0 +1,42 @@
+// This class is object representation of data response to clined
+// side on /auth endpoint
+
+package com.google.sps.data;
+
+
+public class AuthData {
+    
+    private final boolean isUserLoggedIn;
+    private final String nickname;
+    private final String loginUrl;
+    private final String logoutUrl;
+    private final String userId;
+
+    public AuthData(boolean isUserLoggedIn, String nickname, String loginUrl, String logoutUrl, String userId){
+        this.isUserLoggedIn = isUserLoggedIn;
+        this.nickname = nickname;
+        this.loginUrl = loginUrl;
+        this.logoutUrl = logoutUrl;
+        this.userId = userId;
```

Comment:
Is this email of the user or an auto-generated ID ?

Reply:
Quote copied from: [https://cloud.google.com/appengine/docs/standard/java/users](url)
> The User service API can return the current user's information as a User object. Although User objects can be stored as a property value in the datastore, we strongly recommend that you avoid doing so because this includes the email address along with the user's unique ID. If a user changes their email address and you compare their old, stored User to the new User value, they won't match. Instead, consider using the User user ID value as the user's stable unique identifier.
 
According to documentation I think ID is something that uniquely identifies user. It is suggested to use ID as identifier instead of email as email address can sometimes be changed.

Reply:
In that case, let us also have an email address.

Reply:
ok, added.